<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangePwdDialog</class>
 <widget class="QDialog" name="ChangePwdDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>197</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Change Password</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background:  rgba(32, 80, 96, 100);
	color: white;
}

QLabel, QCheckBox
{ 
	color: white; 
	border-radius: 3px;
}


QPushButton, QComboBox
{
	color: white;
	background-color: #27a9e3;
	border-width: 0px;
	border-radius: 3px;
}

QPushButton:hover, QComboBox:hover, QPushButton:checked{ background-color: #66c011; }

QTableView
{
	background:  rgba(32, 80, 96, 100);
	color: white;
}

QHeaderView::section { color:black }</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLineEdit" name="pwdLineEdit_1">
     <property name="font">
      <font>
       <family>微軟正黑體</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="maxLength">
      <number>12</number>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_1">
     <property name="font">
      <font>
       <family>微軟正黑體</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Enter New Password</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <family>微軟正黑體</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Enter Password Again</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="pwdLineEdit_2">
     <property name="font">
      <font>
       <family>微軟正黑體</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="maxLength">
      <number>12</number>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="showPwCheckBox">
     <property name="font">
      <font>
       <family>微軟正黑體</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Show Password</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="confirmBtn">
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Confirm</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBtn">
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
